docs/installation.md

```diff
@@ -28,12 +28,36 @@ Run the command:
 docker build -t caicloud/ciao .
 ```
 
-### Install Dockerized kernel
+### Install [nb2kg](https://github.com/jupyter/nb2kg)
+
+Jupyter Notebook Extension to Kernel Gateway (nb2kg) is used to communicate with the kernel in the docker image, please install the package using the command:
+
+```
+pip install nb2kg
+```
+
+Then enable it:
+
+```
+jupyter serverextension enable --py nb2kg --sys-prefix
+```
+
+### Run the kernel
 
 Run the command:
 
 ```
-./hack/install-dockerized.sh
+docker run -v {kubeconfig}:{kubeconfig} -p 8889:8889 caicloud/ciao
+```
+
+Then you could run the notebook with nb2kg extension:
+
+```
+export KG_URL=http://127.0.0.1:8889
+jupyter notebook \
+  --NotebookApp.session_manager_class=nb2kg.managers.SessionManager \
+  --NotebookApp.kernel_manager_class=nb2kg.managers.RemoteKernelManager \
+  --NotebookApp.kernel_spec_manager_class=nb2kg.managers.RemoteKernelSpecManager
```

Comment:
The installation guide becomes a bit complex now, I'd suggest at least explain what does it mean by running `Dockerized Kernel` (or `kuberized kernel`) vs Native kernel, and give a bit introduction.

Reply:
Added a section in the design doc